components/EventsComponent.vue

```diff
@@ -18,17 +18,31 @@ const d = Object.fromEntries(Object.entries(data).filter(([k, v]) =>
 ).sort())
 </script>
 
+<style scoped>
+.eventsList h3:first-of-type {
+  border-top-width: 0;
+  padding-top: 0;
+}
+.eventsList h3 {
+  border-top: 1px solid var(--vp-c-divider);
+  margin-top: 18px;
+  padding-top: 18px;
+}
+.eventsList :deep(table p) {
+  margin: 0;
+}
+</style>
+
 <template>
- <template v-for="(v, k) in d">
-  <h3 :id="k" tabindex="-1">
-   <code>{{ k }}</code>
-   <a class="header-anchor" :href="'#' + k"></a>
-  </h3>
+ <span class="eventsList">
+  <template v-for="(v, k) in d">
```

Comment:
This could semantically be a `<article>` instead of a `<template>`.
And then the styles need to be
```css
.eventsList article:first-of-type {
  /* ... */
}
.eventsList article {
  /* ... */
}
```

Reply:
This is incorrect.  Template tag is handled specially by Vue (see https://vuejs.org/api/built-in-special-elements#template) - we specifically don't want anything added to DOM output, we are just using it as a placeholder tag so we can use Vue to loop through the events variable.

We can't do it in the "eventsList" tag, because that's the parent container (we need a parent container so we can hide the initial border, with the first-of-type CSS).

We can't do it in the h3 tag because the loop is run AFTER the h3 is output to the page, which is not what we want (we want an h3 tag for each event).

Reply:
I meant:
```html
<section class="eventsList">
  <article v-for="(v, k) in d">
    <!-- ... -->
  </article>
</section>
```

This would render a new container element, that contains each `<h3>` and content. And according to [mdn](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article): "The `<article>` HTML element represents a self-contained composition in a document, page, application, or site, which is intended to be independently distributable or reusable (e.g., in syndication)." From a semantic standpoint it would make sense to encapsulate it.

Reply:
OK, I will trust your superior HTML semantic knowledge.  (In my defense, it does require slightly more CSS with this additional element to get the margin spacing correct...)

Reply:
Was you supposed to put the article encapsulation here?

Reply:
For the sake of getting this PR forward I would waiver my request for semantic improvement for now. I doubt we have complete HTML5 compliancy and semantic soundness everywhere, anyway.

If we ever consider improving the screenreader support of our docs, then we can tackle all of these issues consolidatedly. So I will approve as it is now.